cypress/private/versions.bzl

```diff
@@ -1,34 +1,41 @@
 """Integrity hashes for cypress versions"""
 
-# Use /scripts/mirror_release.sh to add a newer version below.
+# Use ./scripts/mirror_releases.sh to add a newer version below.
 # Versions should be descending order so TOOL_VERSIONS.keys()[0] is the latest version.
 TOOL_VERSIONS = {
+    "13.5.0": {
+        "darwin-x64": "c03869bdd0f1eca1aa6b34f3529a3a66f3c3f37466ab70b951df9e380ed0e0c7",
+        "darwin-arm64": "d1fe007cbe799ab614a46079c33d22cdf759c5f33f3a759b13b0948a28aa0159",
+        "linux-x64": "d1fba34ba408514506c162be8041dfeb37ac42f777ec0e369c58cd0edbcaa000",
+        "linux-arm64": "edd4d76f0ab4228d7207ad57aa4c08a2a60528d056a6ea0397ae3f1c4d365d51",
+        "win32-x64": "35c157342b8ae120e1e37e519d255d60e5bec3634bb9c488cda161ece24cc4e5",
+    },
     "13.3.1": {
         "darwin-x64": "417c5f1d77e15c8aef0a55f155c16c3dbbc637f918c1e51f8fec6eb1c73a9ba9",
         "darwin-arm64": "143d905779c0b0a8a9049b0eb68b4b156db3d838d4546ce5082a8f7bd5dc5232",
-        "linux-x64": "a165aa290d23d03f191852df2e8c3cb25ddccf1d4e226518a0a217f4ec405328",
-        "linux-arm64": "b51810d7fda748f67d5d875d5e265ee22bb394af2bb4a545b78c99f1aae91cb0",
+        "linux-x64": "bb0ddd980bd82792a477b1c39ce8a0a7b275481031c559c065c94f1131151b0c",
```

Comment:
Why did these change for old versions?

Reply:
They changed upstream so I re-generated all of the hashes